src/wallet/api/wallet.cpp

```diff
@@ -163,12 +163,11 @@ struct Wallet2CallbackImpl : public tools::i_wallet2_callback
         }
     }
 
-    virtual void on_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount, const cryptonote::subaddress_index& subaddr_index, uint64_t unlock_time)
+    virtual void on_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount, const cryptonote::subaddress_index& subaddr_index, uint64_t unlock_time, bool blink)
```

Comment:
These methods should all be changed `override` (and not `virtual`).  It's just bad code to have `virtual` (and not `override`!) in an overridden derived class method definition.

Reply:
Updated